libs/domain/search/facet/facet.component.ts

```diff
@@ -0,0 +1,115 @@
+import { FacetValue } from '@spryker-oryx/product';
+import {
+  computed,
+  i18n,
+  signalAware,
+  signalProperty,
+} from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { when } from 'lit/directives/when.js';
+import { FacetController } from './controllers';
+import { searchFacetStyles } from './facet.styles';
+
+@signalAware()
+export class SearchFacetComponent extends LitElement {
+  static styles = [searchFacetStyles];
+
+  protected controller = new FacetController(this);
+
+  @signalProperty() name?: string;
+  @signalProperty({ type: Boolean }) open = false;
+  @signalProperty({ type: Boolean }) multi = false;
+  @signalProperty({ type: Number }) renderLimit = 5;
+  @signalProperty({ type: Number }) minForSearch = 13;
+  @signalProperty({ type: Boolean }) enableClear = true;
+
+  protected facet = computed(() => this.controller.getFacet());
+
+  protected onChange(e: InputEvent): void {
+    const { value, checked: selected } = e.target as HTMLInputElement;
+    this.controller.dispatchSelectEvent({
+      value,
+      selected,
+    });
+  }
```

Comment:
I think we should move the method back to component

Reply:
It is no the best idea due to controller also uses this dispatcher inside onClear event handler. If we move to the component then we should duplicate the dispatcher for the controller or make it public and create unnucessary circular dependency between controller and componen